### ISSUE TITLE
ssd_mobilenet_v2 frozen model size much larger than ssd_mobilenet_v1

### ISSUE THREAD

**Issue by enochkid** [tensorflow/tensorflow#22233]
Hi, comparing the pre-trained models published on tensorflow object detection model zoo, why the frozen_model/check_point/saved_model of ssd_mobilenet_v2_coco is much larger than the ones of ssd_mobilenet_v1_coco? mobilenet v2 should be more lite than mobilenet v1.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce
Mobile device

**Comment by tensorflowbutler** [comment]
Nagging Assignee @achowdhery: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @achowdhery: It has been 15 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @shashishekhar: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @shashishekhar: It has been 29 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by achowdhery** [comment]
Please try using ssdlite for mobilenet v2: http://download.tensorflow.org/models/object_detection/ssdlite_mobilenet_v2_coco_2018_05_09.tar.gz

**Comment by ymodak** [comment]
Closing this issue since its resolved. Feel free to reopen if have any further problems. Thanks!
